recipes/fsleyes_props/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "fsleyes-props" %}
+{% set version = "1.6.3" %}
+{% set sha256 = "df5bac8e73da060754a4500ccb485bb4b97738381b2c56ab9bcd5fca0c56599a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
```

Comment:
You can just remove this build section here. That should work fine without it  :)

Reply:
Removing the `requirements/build` section caused the buids to fail, e.g.:

https://travis-ci.org/conda-forge/staged-recipes/builds/424841283?utm_source=github_status&utm_medium=notification

Any ideas?

Reply:
Actually, it looks like something else has caused the builds to fail - adding the section back in didn't fix them ...

Reply:
The most glaring difference between an earlier build which succeeded:

https://travis-ci.org/conda-forge/staged-recipes/builds/423823890?utm_source=github_status&utm_medium=notification

and a subsequent one which failed (identical recipe for both builds):

https://travis-ci.org/conda-forge/staged-recipes/builds/424860749?utm_source=github_status&utm_medium=notification

is an update of `conda-build` from 3.13.0 to 3.14.0. Could something have been broken?

Reply:
It is, but it looks like it just couldn't connect to anaconda.org. If you need you can restart the build (or you can ask for help restarting the build.)

Reply:
@scopatz thanks for getting back to me - are you sure about that? The builds are failing on a HTTP request to local files (e.g.  `file:///Users/travis/miniconda/conda-bld/noarch/repodata.json` on the travis builds).

And it looks like, at least on travis, *all* PR builds have failed since conda 3.14.0 was made available - the last successful one ([PR #6126, about 7 hours ago](https://travis-ci.org/conda-forge/staged-recipes/builds/424698463)) used conda 3.13.0.

Reply:
I am not sure about that. The correct person to flag is probably @msarahan

Reply:
This is related to
https://github.com/conda/conda-build/issues/3114

And I opened a PR in ``staged-recipes`` to do a work around for now
https://github.com/conda-forge/staged-recipes/pull/6627

Reply:
@marcelotrevisani Great, thanks for chasing that up!